TypeCobol.LanguageServer.Test/LSRTests/MultiLineCommentIncScan/input/MultiLineCommentIncScan.tlsp

```diff
@@ -62,43 +62,7 @@
-    },
-    {
-      "category": 1,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM52032788062027908163.cee\",\"diagnostics\":[{\"range\":{\"start\":{\"line\":33,\"character\":8},\"end\":{\"line\":33,\"character\":8}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : no viable alternative at input '%<<< ... >'\"},{\"range\":{\"start\":{\"line\":23,\"character\":12},\"end\":{\"line\":23,\"character\":43}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : extraneous input 'DECLAREPROCEDUREMyProcpublic.' expecting {DeclarativesHeader, SectionHeader, ParagraphHeader, SentenceEnd, AcceptStatement, AddStatement, AllocateStatement, AlterStatement, CallStatement, CancelStatement, CloseStatement, ComputeStatement, ContinueStatement, DeleteStatement, DisplayStatement, DivideStatement, EntryStatement, EvaluateStatement, ExecStatement, ExitMethodStatement, ExitProgramStatement, ExitStatement, FreeStatement, GobackStatement, GotoStatement, IfStatement, InitializeStatement, InspectStatement, InvokeStatement, JsonGenerateStatement, JsonParseStatement, MergeStatement, MoveStatement, MultiplyStatement, OpenStatement, PerformProcedureStatement, PerformStatement, ReadStatement, ReleaseStatement, ReturnStatement, RewriteStatement, SearchStatement, SetStatement, SortStatement, StartStatement, StopStatement, StringStatement, SubtractStatement, UnstringStatement, WriteStatement, XmlGenerateStatement, XmlParseStatement, FunctionDeclarationEnd, ProcedureStyleCall}\"},{\"range\":{\"start\":{\"line\":4,\"character\":8},\"end\":{\"line\":4,\"character\":28}},\"severity\":2,\"code\":\"37\",\"source\":\"\",\"message\":\"Warning: \\\"END PROGRAM\\\" is missing.\"}]}}"
-    },
-    {
-      "category": 0,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/didChange\",\"params\":{\"textDocument\":{\"version\":0,\"uri\":\"file:/C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM52032788062027908163.cee\"},\"uri\":\"file:/C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM52032788062027908163.cee\",\"contentChanges\":[{\"range\":{\"start\":{\"line\":29,\"character\":9},\"end\":{\"line\":29,\"character\":10}},\"rangeLength\":1,\"text\":\"\"}]}}"
-    },
-    {
-      "category": 1,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM52032788062027908163.cee\",\"diagnostics\":[]}}"
```

Comment:
What happened to these steps in the original LSR test ?

Reply:
fixed.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/4b5e1d0af58323248efbd75657d4b554eb7db081